### PR TITLE
Add new mobile phone event limitation

### PR DESCRIPTION
**Pull request by robertnicolo-okta**
## Description:
Add new mobile phone event limitation

### Resolves:

* [OKTA-526539](https://oktainc.atlassian.net/browse/OKTA-526539)

### Deploy link to new limitation

https://6306492703bf300f26eec1bb--reverent-murdock-829d24.netlify.app/docs/guides/ie-limitations/main/#target-id-value-simulated-for-mobile-phone-event-types

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md [hunk 1]
```diff
@@ -44,6 +44,36 @@ The following Event Types are available only in Identity Engine and can't be use
...
 ***
 
+### Target.ID value simulated for mobile phone event types
+
+**What Changed:** `Target.ID` represents a simulated value for the following event types:
+
+* `system.voice.send_phone_verification_call`
+* `system.sms.send_phone_verification_message`
+
+Specifically, when `Target.Type` equals `MobilePhone` for these events, `Target.ID` has no intrinsic value other than being a placeholder that allows backward compatibility with Classic. In Classic, `Target.ID` is unique per phone number, stored in Okta's backend systems, and can be used to link all events for a given phone number. The following example from the System Log API shows these fields:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md:54]:
```suggestion
Specifically, when `Target.Type` equals `MobilePhone` for these events, `Target.ID` has no intrinsic value other than being a placeholder that allows backward compatibility with Classic Engine. In Classic Engine, `Target.ID` is unique per phone number, stored in Okta's backend systems, and can be used to link all events for a given phone number. The following example from the System Log API shows these fields:
```

File: packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md [hunk 2]
```diff
@@ -44,6 +44,36 @@ The following Event Types are available only in Identity Engine and can't be use
...
+
+```json
+  "target": [
+    {
+      "id": "qFGtyCw3hwhHh55555555z",
+      "type": "MobilePhone",
+      "alternateId": +15555555555,
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md:61]:
```suggestion
      "alternateId": "+15555555555",
```

File: packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md [hunk 3]
```diff
@@ -44,6 +44,36 @@ The following Event Types are available only in Identity Engine and can't be use
...
+      "displayName": +15555555555,
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md:62]:
```suggestion
      "displayName": "+15555555555",
```

**Review by susanharper-okta — APPROVED** [review]
couple nits. approving to not hold you up.

File: packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md (hunk 1 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md]: Updated in latest commit.

File: packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md (hunk 2 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/ie-limitations/main/index.md]: updated in latest commit.

**Pull request merged**